app/helpers/application_helper/toolbar/host_initiator_groups_center.rb

```diff
@@ -16,7 +16,7 @@ class ApplicationHelper::Toolbar::HostInitiatorGroupsCenter < ApplicationHelper:
             :image   => "refresh",
             :data    => {'function'      => 'sendDataWithRx',
                          'function-data' => {:type => "refresh", :controller => "hostInitiatorGroupToolbarController"}},
-            :confirm => N_("Refresh relationships and power states for all items related to these Host Initiators Cluster?"),
+            :confirm => N_("Refresh relationships and power states for all items related to these Host Initiators Clusters?"),
```

Comment:
Assuming the singular is `Host Initiator Cluster` the plural will be `Host Initiator Clusters` (just making sure this isn't something plural-possessive like `Host Initiators' Clusters`